core/templates/components/state-directives/response-header/response-header.component.html

```diff
@@ -42,26 +42,7 @@
   </span>
 </div>
 
-<span class="oppia-delete-response-button oppia-transition-200 e2e-test-delete-response"
```

Comment:
@Lawful2002 As we are using response-header  component  many places in the code base 
Due to the changes of this PR we lost the cross button 
that is used to delete the button.
And this arise a issue  
See:
https://github.com/oppia/oppia/issues/16948
Can you please help me to fix this.
Thanks.